### ISSUE TITLE
Cannot find namespace 'debug' in 2.29.0

### ISSUE THREAD

**Issue by millsp** [prisma/engines-wrapper#128]
```
+ pnpm recursive run build
Scope: all 5 workspace projects
packages/engines-version build$ tsc -d
packages/get-platform build$ tsc -d
packages/engines-version build: Done
packages/get-platform build: Done
packages/fetch-engine build$ tsc -d
Error: packages/fetch-engine build: ../../node_modules/.pnpm/@prisma+debug@2.29.0/node_modules/@prisma/debug/dist/index.d.ts(2,44): error TS2503: Cannot find namespace 'debug'.
packages/fetch-engine build: Failed
undefined
/home/runner/work/engines-wrapper/engines-wrapper/packages/fetch-engine:
 ERROR  @prisma/fetch-engine@2.29.0-34.1be4cd60b89afa04b192acb1ef47758a39810f3a build: `tsc -d`
Exit status 2
```

**Comment by Jolg42** [comment]
Before (2.28) https://unpkg.com/browse/@prisma/debug@2.28.0/dist/index.d.ts
```
import DebugLib from 'debug';
declare function Debug(namespace: string): DebugLib.Debugger;
declare namespace Debug {
    var enable: (namespace: string) => void;
    var enabled: (namespace: string) => boolean;
}
export default Debug;
export { Debug };
export declare type Debugger = DebugLib.Debugger;
export declare function getLogs(numChars?: number): string;
```

Now (2.29) https://unpkg.com/browse/@prisma/debug@2.29.0/dist/index.d.ts
```
import 'debug';
declare function Debug(namespace: string): debug.Debugger;
declare namespace Debug {
    var enable: (namespace: string) => void;
    var enabled: (namespace: string) => boolean;
}
export default Debug;
export { Debug };
export declare function getLogs(numChars?: number): string;
```

**Comment by Jolg42** [comment]
Fix now available in the `2.29.1` patch just released a minute ago.
